.github/workflows/terraform_provider.yml

```diff
@@ -0,0 +1,258 @@
+name: Terraform Provider Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - .github/workflows/terraform_provider.yml
+      - .golangci.yml
+      - aws/**
+      - awsproviderlint/**
+      - docs/index.md
+      - docs/data-sources/**
+      - docs/guides/**
+      - docs/resources/**
+      - go.sum
+      - GNUmakefile
+      - main.go
+      - staticcheck.conf
+      - website/**
+
+env:
+  AWS_DEFAULT_REGION: us-west-2
+  GO_VERSION: "1.14"
```

Comment:
Should `.go-version` be updated to a 1.14 as well?

Reply:
At this point, probably yes! I've been using 1.14.0 through 1.14.2 just fine locally the last few weeks. There's a few things that need to be updated when we do that so let's create a separate tracking issue and ensure the maintaining guide has all those items listed. 👍